### ISSUE TITLE
No indentation error for indented first line (CPython gives an error)

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [microsoft/pyright#2816]
**Describe the bug**

Pyright gives no error for this document:

```python
    pass
pass
```

but Python reports an IndentationError on line 1. 

The second example in the [indentation section of the lexical analysis docs](https://docs.python.org/3/reference/lexical_analysis.html#indentation) calls this out as an error though I'm not clear on where it's supported by the surrounding text.

This isn't very likely bug for experienced programmers to hit but can affect new programmers copy/pasting small scripts from docs.

**To Reproduce**

```bash
➜  tmp npm i pyright
...
+ pyright@1.1.203
added 1 package from 1 contributor and audited 1 package in 3.283s
found 0 vulnerabilities

➜  tmp ./node_modules/.bin/pyright --version
pyright 1.1.203
➜  tmp echo "    pass\npass\n" > test.py    
➜  tmp ./node_modules/.bin/pyright
No configuration file found.
No pyproject.toml file found.
stubPath /Users/mth/tmp/typings is not a valid directory.
Assuming Python platform Darwin
Searching for source files
Found 1 source file
0 errors, 0 warnings, 0 infos 
Completed in 0.538sec
➜  tmp python3 test.py 
  File "/Users/mth/tmp/test.py", line 1
    pass
IndentationError: unexpected indent
```

**Expected behavior**
Error similar to Python's.

**Screenshots or Code**
See above.

**VS Code extension or command-line**
pyright@1.1.203 as above though I can also see this in Pylance v2021.12.2.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.205, which I just published. It will also be included in the next release of pylance.
